src/lib/components/events/event-item.svelte

```diff
@@ -21,6 +21,13 @@
       .catch(() => toast({ path, type: ToastType.Error, content: errorMessage }));
   }
 
+  function formatLocation(location?: string | null, hosted = ['Discord', 'Zoom']): string {
+    // '', null, and undefined are all TBD
```

Comment:
can we get a unit test on this function for all 3 of these cases + the 2 base cases?

Reply:
test cases for all TBD and online locations added!